src/Implementation/Formatters/UniqueItems.php

```diff
@@ -10,8 +10,16 @@ class UniqueItems extends Formatter
 
     public function replacements(): array
     {
```

Comment:
Since json_encode of a a string would be a string I suggest to simplify if-else to:
```suggestion
        $duplicate = $this->error->keywordArgs()['duplicate'];
        $duplicate = json_encode($duplicate);
        
        return [
            ':duplicate:' => $duplicate
        ];
```